style.css

```diff
@@ -238,42 +533,6 @@ ul {
     margin-bottom: -200px;
   }
 
-  .aboutme-button {
```

Comment:
[Optional] It would also be nice to add the `cursor: pointer` style to all your buttons so that when a user hovers over a button, their cursor changes to a pointer✨. This helps improve user experience, so they don't have to guess whether a button is a pointer or not😉

Reply:
Fixed.